src/main/java/bhs/devilbotz/Constants.java

```diff
@@ -7,64 +7,70 @@
 import edu.wpi.first.wpilibj.RobotBase;
 
 public class Constants {
-    // How often to run a loop of the robot
-    // Default is 20ms (do not change unless you know what you're doing
-    public static final double loopPeriodSecs = 0.02;
-    // Tuning mode toggle
-    public static final boolean tuningMode = false;
-    // Which robot to use?
-    // Options:
-    // - ROBOT_2024S
-    // - ROBOT_2024SIM
-    private static final RobotType robot = RobotType.ROBOT_2024SIM;
+  // How often to run a loop of the robot
+  // Default is 20ms (do not change unless you know what you're doing
+  public static final double loopPeriodSecs = 0.02;
+  // Tuning mode toggle
+  public static final boolean tuningMode = true;
```

Comment:
Should tuningMode be committed as a default?

Reply:
no, let me fix that now.
```suggestion
  public static final boolean tuningMode = false;
```